app/ActivityListeners/ActivityListener.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: macintosh
+ * Date: 6/15/17
+ * Time: 8:40 AM
+ */
+
+namespace App\ActivityListeners;
+
+
+use App\Models\Activity;
+use App\Models\ActivityNotification;
+
+class ActivityListener
+{
+    public static $handlerClasses = [
+        CopyPostHandler::class,
+        EditPostHandler::class,
+        CreatePostHandler::class,
+        ChangeTargetSiteHandler::class,
+        DuplicateTitleHandler::class,
+        FailedAutosaveHandler::class,
+        FailedPostToTargetSite::class,
+        PostLockHandler::class,
+        DevToolEventHandler::class
+    ];
+
+    public static function handle(Activity $activity) {
+        $messages = [];
+
+        foreach(self::$handlerClasses as $class) {
+            /** @var BaseHandler $class */
+            $events = $class::event();
+            if(!is_array($events)) {
+                $events = [$events];
+            }
+            foreach($events as $event) {
+                if($event != $activity->action) {
+                    continue;
+                }
+
+                $message = (new $class())->handle($activity);
+                if($message != false) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1125)